locale/es/docs/guides/security/index.md

```diff
@@ -0,0 +1,334 @@
+
+**Mitigaciones**
+
+* La API criptográfica expone una función `timingSafeEqual` para comparar valores sensibles reales y valores sensibles esperados utilizando un algoritmo de tiempo constante.
+* Para la comparación de contraseñas, se puede utilizar la función [scrypt][] disponible también en el módulo criptográfico nativo.
+* De forma más general, evitar el uso de secretos en operaciones de tiempo variable. Esto incluye bifurcarse en secretos y, cuando el atacante podría estar ubicado en la misma infraestructura (por ejemplo, la misma máquina en la nube), utilizar un secreto como índice en la memoria. Escribir código de tiempo constante en JavaScript es difícil (en parte debido al JIT). Para las aplicaciones criptográficas, utilice las API criptográficas integradas o WebAssembly (para los algoritmos no implementados de forma nativa).
+
+### Módulos maliciosos de terceros (CWE-1357)
+
+Actualmente, en Node.js, cualquier paquete puede acceder a recursos potentes como acceso a la red. Además, como también tienen acceso al sistema de archivos, pueden enviar cualquier dato a cualquier lugar.
+
+Todo el código que se ejecuta en un proceso de Node.js tiene la capacidad de cargar y ejecutar código arbitrario adicional utilizando `eval()` o sus equivalentes.
+Todo el código con acceso de escritura al sistema de archivos puede lograr lo mismo, escribiendo en archivos nuevos o existentes que se han cargado.
+
+Node.js tiene un [policy mechanism][] que es experimental[¹][experimental-features] para declarar el recurso cargado como no fiable o fiable. Sin embargo, esta política no está habilitada por defecto.
+
+Asegúrate de fijar las versiones de las dependencias y de ejecutar comprobaciones automáticas de vulnerabilidades utilizando flujos de trabajo comunes o através de scripts de Npm.
+
+Antes de instalar un paquete asegúrese de que éste esta mantenido y que incluye todo el contenido que esperaba. Tenga cuidado, ya que el código fuente en Github no siempre es el mismo que el publicado en Npm, valídelo en el _node\_modules_.
+
+#### Ataques a la cadena de suministro
+
+Un ataque a la cadena de suministro en una aplicación Node.js, ocurre cuando una de sus dependencias (ya sea directa o transitiva) está comprometida. Esto puede ocurrir debido a que la aplicación es demasiado laxa en la especificación de las dependencias (permitiendo actualizaciones no deseadas) y/o errores comunes en la especificación (vulnerable a [typosquatting][]).
```

Comment:
**suggestion:** `la aplicación es demasiado laxa` --> `la aplicación no es estricta`

Reply:
Fixed in [c4619df](https://github.com/nodejs/nodejs.org/pull/4932/commits/c4619df7c402460da9dff8c71ce9e3205de81181)